api/data_overview_builder.py

```diff
@@ -0,0 +1,125 @@
+"""
+Data Overiew Builder generates a markdown file with information about data in the API.
+
+By default this is shown on the docs site.
+
+"""
+from typing import List
+import dataclasses
+
+import pydantic
+
+from api import can_api_v2_definition
+
+Actuals = can_api_v2_definition.Actuals
+Metrics = can_api_v2_definition.Metrics
+RiskLevels = can_api_v2_definition.RiskLevels
+
+
+@dataclasses.dataclass
+class Fields:
+    actuals: List[str]
+    metrics: List[str]
+
+
+CASES_FIELDS = Fields(["cases", "newCases"], ["caseDensity", "infectionRate", "infectionRateCI90"],)
+
+DEATHS_FIELDS = Fields(["deaths"], [])
+
+HOSPITALIZATIONS_FIELDS = Fields(
+    ["icuBeds", "hospitalBeds"], ["icuCapacityRatio", "icuHeadroomRatio"],
+)
+
+VACCINATION_FIELDS = Fields(
+    ["vaccinesDistributed", "vaccinationsInitiated", "vaccinationsCompleted"],
+    ["vaccinationsInitiatedRatio", "vaccinationsCompletedRatio"],
+)
+
+
+TESTING_FIELDS = Fields(
+    ["positiveTests", "negativeTests"], ["testPositivityRatio", "testPositivityRatioDetails"],
+)
+
+
+@dataclasses.dataclass
+class Section:
+
+    name: str
+    fields: List[pydantic.Field]
+    description: str = ""
+
+
+CASE_DESCRIPTION = """
+
+
+"""
```

Comment:
WIP? CASE_DESCRIPTION is empty and will be more readable with new lines after it.